prow/config.yaml

```diff
@@ -1577,6 +1577,92 @@ postsubmits:
           - name: cache-ssd
             hostPath:
               path: /mnt/disks/ssd0
+      - name: ci-kubernetes-e2e-kubeadm-gce-upgrade-1-6-1-7
```

Comment:
this should after this one: https://github.com/kubernetes/test-infra/blob/master/prow/config.yaml#L16206